com.sap.cloud.lm.sl.cf.core/src/main/java/com/sap/cloud/lm/sl/cf/core/util/SSLUtil.java

```diff
@@ -32,7 +32,7 @@ public void checkServerTrusted(X509Certificate[] xcs, String string) {
 
         @Override
         public X509Certificate[] getAcceptedIssuers() {
-            return null;
```

Comment:
Are you sure null and an empty array are equivalent in this context?

Reply:
Yes, in the implementation of the SSLContext upon initiating the engine, when the trust manager is being chosen, there is a check for null and, if not, iterating through the array, etc. But in our case, both null and an empty array result in a dummy X509 trust manager. So, the functionality isn't being changed.